index.js

```diff
@@ -50,21 +78,43 @@ class Aperture {
           reject(new Error('Invalid `cropArea` option object'));
           return;
         }
-
-        cropArea = `${cropArea.x}:${cropArea.y}:${cropArea.width}:${cropArea.height}`;
       }
 
-      const recorderOpts = [
-        this.tmpPath,
-        fps,
-        cropArea,
-        showCursor,
-        highlightClicks,
-        displayId,
-        audioSourceId
-      ];
+      if (IS_MACOS) {
+        const recorderOpts = [
+          this.tmpPath,
+          fps,
+          cropArea ? `${cropArea.x}:${cropArea.y}:${cropArea.width}:${cropArea.height}` : 'none',
+          showCursor,
+          highlightClicks,
+          displayId,
+          audioSourceId
+        ];
+
+        this.recorder = execa(path.join(__dirname, 'swift', 'main'), recorderOpts);
+      } else if (IS_WINDOWS) {
+        const ffmpegArgs = [];
+
+        if (typeof cropArea === 'object') {
+          ffmpegArgs.push(
+            '-video_size', `${cropArea.width}x${cropArea.height}`,
+            '-f', 'gdigrab',
+            '-i', 'desktop',
```

Comment:
The above two arguments can be in the initial array as they appear the same in both conditions.